zntrack/core/base.py

```diff
@@ -698,33 +698,35 @@ def operating_directory(
         move_on: EXCEPTION_OR_LST_EXCEPTIONS = Exception,
         disable: bool = None,
     ) -> bool:
-        """Work in an operating directory until successfully finished.
-
-        This context manager will replace $nwd$ with 'prefix_$nwd$' and move the files
-        to $nwd$ when successfully finished. This can be useful, when you are running
-        e.g., on hardware with limited execution time and can't use 'dvc checkpoints'.
-        When successfully finished, all files will be moved from 'temp_$nwd$' to $nwd$.
-        You can call 'dvc repro' multiple times to continue from 'temp_$nwd$'.
-        If used properly this will result in reproducible data but:
+        """Work in an temporary operating directory until successfully finished.
+
+        This context manager will replace $nwd$ with a temporary operating
+        directory 'prefix_$nwd$' and move the files to $nwd$, when successfully
+        finished. This can be useful, when you are running, e.g., on hardware
+        with limited execution time and can't use 'dvc checkpoints'.
+        When successfully finished, all files will be moved from 'prefix_$nwd$' to $nwd$.
+        You can call 'dvc repro' multiple times to continue from 'prefix_$nwd$'.
+        If used properly this will result in reproducible data, but:
         - checkpoints will not be removed if parameters change. Always remove a
             checkpoint, when running with new parameters!
-        - checkpoints are not versioned. If you want to checkpoint e.g., model training,
+        - checkpoints are not versioned. If you want to checkpoint, e.g., model training,
             use 'dvc checkpoints'.
 
         Parameters
         ----------
         prefix: str, default = 'ckpt'
             Prefix for the temporary directory
         remove_on: Exception or list of Exceptions, default = None
-            If one of the exceptions in 'remove_on' is raised, the operating directory
-             will be removed. Otherwise, it will remain and reused upton the next run.
+            If one of the exceptions in 'remove_on' is raised, the temporary
+            operating directory
+            will be removed. Otherwise, it will remain and reused upon the next run.
         move_on: Exception, default = Exception
-            If one of the exceptions in 'move_on' is raised, the operating directories
-            content is moved to $nwd$ and the directory will be removed.
-            This helps, in the case of an error to not restart from an already
-            failed data point.
+            If one of the exceptions in 'move_on' is raised, the temporary
```

Comment:
Maybe we add a section: The default behavior will move the files upon any exception.